tests/third_party/cupy/testing/_random.py

```diff
@@ -94,12 +94,12 @@ def fix_random():
     """Decorator that fixes random numbers in a test.
 
     This decorator can be applied to either a test case class or a test method.
-    It should not be applied within ``_condition.retry`` or
-    ``_condition.repeat``.
+    It should not be applied within ``condition.retry`` or
```

Comment:
What is the reason for this renaming?
We have `_condition,py` instead of `condition.py` file

Reply:
It is aligned with cupy. I would like to avoid additional lines of diff in base f/w between our snapshot and what is in cupy code.